content/en/guides/employee-engagement.md

```diff
@@ -2,198 +2,201 @@
+You should avoid proposing large changes to the project’s code base or documentation as a first act of participation. You need to build out a good relationship and trust with the project’s contributors first. This will help you to align your future contributions with the project’s overall mission and goals.
 
-While many open source projects have good documentation and active communication channels to learn from, you might still have questions that you haven’t found the answers for. Asking questions on the mailing list or chat platform is a good way to get in touch with the community. Always make sure that you communicate clearly by sharing background to put your question or comment into context.
+While many open source projects have good documentation and active communication channels to learn from, you might still have questions that you haven’t found the answers for. Asking questions on the mailing list or chat platform is a good way to get in touch with the community. Always make sure that you communicate clearly by sharing background information so that your question or comment has the necessary context.
 
-If you spent a little bit of time deploying and trying out the project that you have been looking into, you already built up some good knowledge. Even as a newbie you might know the answer to somebody else’s question, or can contribute to help them figuring out a next step in what they are trying to do. Don’t be shy to chime in to the conversation and help somebody else.
+If you have spent a little bit of time deploying and trying out the project you will already have some good knowledge. Even as a newbie you might know the answer to somebody else’s question, or can contribute to help them figuring out a next step in what they are trying to do. Don’t be afraid to add to the conversation and help somebody else.
 
-Prioritize to do reviews, code or documentation, because that is one of the best ways to learn about the project while performing a task that is always in need of more help and attention.
+Prioritize giving reviews on code or documentation. People value this activity and it is a good way to learn about the project.
 
-Don’t be afraid to start participating in discussions if you have input to them, either a comment from your experience or a question that can help taking a conversation forward. It is a good way to show the community that you care about their project and would like to participate in maintaining and developing it further.
+Do participate in discussions if you have helpful information to share. Sharing your experience or asking a clarifying question can help move the conversation forward. It is also a good way to show the community that you care about their project and would like to participate in maintaining and developing it further.
```

Comment:
My feeling is that it might be good to tell people to not be afraid. It can be quite intimidating to get involved in a public discussion, maybe for the first time.

Reply:
Thanks, I changed as suggested.